datastore/longhorn.go

```diff
@@ -499,7 +526,7 @@ func (s *DataStore) AreAllVolumesDetached() (bool, error) {
 	return true, nil
 }
 
-func (s *DataStore) AreAllV2VolumesDetached() (bool, error) {
+func (s *DataStore) AreAllVolumesDetached() (bool, error) {
```

Comment:
In `AreAllVolumesDetached`, it filters the pods backed by `longhorn.BackendStoreDriverTypeV2` only?

Reply:
It's wrong. Refactor AreAllVolumesDetached and remove AreAllVolumesDetachedByBackendStoreDriver. Thank you.